src/components/Game/__tests__/onMessage.test.ts

```diff
@@ -283,17 +285,25 @@ describe("walletInfo", () => {
   test("updates the balance and the deposit address", () => {
     const address = "123456789a123456789a123456789a1234";
     const balance = 9.9873;
+    const table_stack_in_chips = 10;
     receiveMessage(
       {
         method: "walletInfo",
         addr: address,
-        balance
+        balance,
+        table_stack_in_chips
       },
       0
     );
 
     expect(updateStateValueSpy).toHaveBeenCalled();
-    expect(updateStateValueSpy).toHaveBeenCalledTimes(2);
+    expect(updateStateValueSpy).toHaveBeenCalledTimes(3);
+    // @todo test keeps on failing because of the below! fix!
```

Comment:
this must be something silly but I decided to postpone fixing it until I got a clear mind